### PR TITLE
utils/cpuload: Print time constant values directly without assigning them to array

### PR DESCRIPTION
**Pull request by jeongarmy**
Define and use static variable for time constant values instead of extern variable, g_cpuload_timeconstant because these values are defined in config.h.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [4be7ed5702b861990d9085d81c39af8ba4b127f4] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4be7ed5702b861990d9085d81c39af8ba4b127f4] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ad899c55421e2f0e51903b851cfc2be2dd73cfa4] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ad899c55421e2f0e51903b851cfc2be2dd73cfa4] - Code Rule Check (C++) OK.

File: apps/system/utils/utils_cpuload.c [hunk 1]
```diff
@@ -96,31 +94,30 @@ static void cpuload_print_tasklist(void)
...
+	printf("%9ds", CONFIG_SCHED_CPULOAD_TIMECONSTANT);
 	printf("\n-----------------\n");
 #endif
 
 	/* Print cpu load for each task */
 	utils_proc_pid_foreach(cpuload_read_proc);
 
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	printf("=====================================\n");
+	printf("==================================\n");
```
Comment (Taejun-Kwon) [inline apps/system/utils/utils_cpuload.c]: It is TOOOOOOOOOOO carefully coded, how about use just only long one? =)

**Comment by seinfra** [comment]
Target : [6d1d4126f96096b01efb91fd8f96875b5b057754] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6d1d4126f96096b01efb91fd8f96875b5b057754] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a2cb529c3ef279e4ed441137128b8192a24cd1fb] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a2cb529c3ef279e4ed441137128b8192a24cd1fb] - Code Rule Check (C++) OK.

**Pull request merged**